connector/src/models/CollateralRequirement.ts

```diff
@@ -0,0 +1,36 @@
+import { formatBn } from '../helpers'
+import { CollateralRequirementData, IHoneypotConnector } from '../types'
+
+export default class CollateralRequirement {
+  #connector: IHoneypotConnector
+
+  readonly id: string
+  readonly voteId: string
```

Comment:
Should this be `proposalId` ?

Think we also forgot to assign it on `constructor`